tests/unit/ground-commander-test.cpp

```diff
@@ -59,11 +64,10 @@ TEST_GROUP(GroundCommanderTestGroup)
 
         // Make sure the commander is running
         while (system("ps aux | grep bin/ground-commander/ground-commander 1>/dev/null") != 0){
-           usleep(1000); 
+           	usleep(2000); 
```

Comment:
Was this change necessary as a result of your testing? Can you write a comment about why this is needed?